CRM/Profile/Selector/Listings.php

```diff
@@ -484,7 +484,7 @@ public function &getRows($action, $offset, $rowCount, $sort, $output = NULL, $ex
     }
     $links = self::links($this->_map, $this->_editLink, $this->_linkToUF, $this->_profileIds);
 
-    $locationTypes = CRM_Core_PseudoConstant::get('CRM_Core_DAO_Address', 'location_type_id', ['labelColumn' => 'name']);
+    $locationTypes = CRM_Core_DAO_Address::buildOptions('location_type_id', 'validate');
```

Comment:
This is slightly different - validate includes disabled, which I personally think is "correct" for validate to do in general, but seems like it would be some kind of change here. I'm not sure what this code is used for - public directory listings? - but the code down below skips it when it's missing.

Reply:
@demeritcowboy looks like it's just used for matching, in which case it's fine to include all options.
Tests pass :)

Reply:
I checked and it does have an effect, but depending on your point of view it could be considered better.

Enabled it looks like this:

![before-enabled](https://github.com/user-attachments/assets/0c4dd777-876e-4b5a-8efe-2303b4fbd4b3)

Disabled, before the patch, it looks like this:

![before-disabled](https://github.com/user-attachments/assets/112dc755-7d78-42b4-ba39-ad148239c880)

Disabled, after the patch, it looks like this:

![after-disabled](https://github.com/user-attachments/assets/16effad0-b222-4aed-a45f-2dca02c9a609)

So on the one hand it fixes a weird display, on the other it's now showing disabled data, which I'm personally fine with but is different from the "standard" way civi does it. But on the third hand, if you've disabled it, why are you still including it as a column in your profile listing. So let's do it.

In both cases with it disabled there's a php error, so it's not a new error: `Warning: Undefined array key 4 in CRM_Profile_Selector_Listings->getColumnHeaders() (line 310 of CRM/Profile/Selector/Listings.php).`

Reply:
Thoroughly tested @demeritcowboy - I agree with your conclusion. The previous behavior wasn't "handling" the disabled-ness it was just an unhandled error causing the display to glitch.

If our change interfered with legit handling that's one thing, but as-is, I agree this is better.